.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,5 +18,6 @@ Please include the following checklist in your PR:
 
 - [ ] The documentation has been updated if necessary.
 - [ ] All code conforms to the [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md), [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md), and [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guides.
-- [ ] All documentation and configuration conforms to the [markdown](https://github.com/enso-org/enso/blob/develop/docs/style-guide/markdown.md) and [YAML](https://github.com/enso-org/enso/blob/develop/docs/style-guide/yaml.md) style guides.
-- [ ] All code has been tested where possible.
+- All code has been tested:
+  - [ ] Unit tests have been written where possible.
+  - [ ] If GUI codebase was changed: BOTH `./run dist` and `./run watch` have been tested.
```

Comment:
```suggestion
  - [ ] If GUI codebase was changed: GUI was tested when built using BOTH `./run dist` and `./run watch`.
```

I would clarify that what we want is to have *the GUI* tested, not just that the `./run watch` and `./run dist` commands succeeded to compile the PR.